lms/djangoapps/course_api/permissions.py

```diff
@@ -0,0 +1,36 @@
+"""
+Course API Authorization functions
+"""
+
+from student.roles import GlobalStaff
+
+
+def can_view_courses_for_username(requesting_user, target_username):
+    """
+    Determine whether `requesting_user` has permission to view courses available
+    to the user identified by `target_username`.  All users can view courses
```

Comment:
Nit: Sentences 2 and 3 are (current) implementation details and not a docstring for the public interface of this method.